packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -448,6 +448,36 @@ export class MetriportMedicalApi {
     return resp.data;
   }
 
+  /**
+   * Associates a patient to multiple facilities.
+   *
+   * @param patientId The ID of the patient to associate facilities to.
+   * @param facilityIds The array of facility IDs to associate with the patient.
+   * @returns The updated facilities associated with the patient.
+   */
+  async assignPatientFacilities(
```

Comment:
"assign" gives the impression it "adds" those facilities to the array of facility IDs.

if we want to have it as an assign/add, then we need a "remove"

or just rename it to "set" and document that it overrides the list of facility Ids?